src/model/mod.rs

```diff
@@ -0,0 +1,3 @@
+mod types;
+
+pub use types::*;
```

Comment:
What does this do ?

Reply:
This is a way of exposing identifiers such that they are easier to access

For example, without this line, the `CellContent` type would be accessible as `crate::model::types::CellContent`,
whereas with this `pub use types::*;`, it makes it available under the name `crate::model::CellContent`.

And thanks to the `*` this will automatically be the case for all public identifiers defined in the `types` module.

I saw that in [this article](https://medium.com/codex/rust-modules-and-project-structure-832404a33e2e)

Reply:
ok ok noice 🫡